solarkraft/test/e2e/tla/.stor/asdf/100/entry-timelock.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
That's cool. I have been thinking about testing storage. It did not occur to me that we could simply save in the tests :))